enterprise/frontend/src/embedding-sdk/cli/steps/ask-tenancy-columns.ts

```diff
@@ -5,13 +5,18 @@ import type { CliStepMethod } from "../types/cli";
 import { printHelperText } from "../utils/print";
 
 export const askForTenancyColumns: CliStepMethod = async state => {
+  // The sample database does not have tenancy columns.
+  if (state.useSampleDatabase) {
+    return [{ type: "success" }, state];
+  }
+
   printHelperText(
     `e.g. does your table have a customer_id column to isolate tenants?`,
   );
 
   const hasTenancyIsolation = await toggle({
     message: `Is your tenancy isolation based on a column?`,
-    default: true,
+    default: false,
```

Comment:
Most users do not have multi-tenancy in their database, so we should default to "no"

Reply:
where has this feedback been raised?

Reply:
@WiNloSt I noticed feedback from [Jeff](https://metaboat.slack.com/archives/C063Q3F1HPF/p1728659005830829) that the CLI doesn't work for him, because he doesn't have tenancy isolation. I noticed that Nicolo also went into tenancy isolation mode without any tables and it bails out, so it's safer to make this optional.

Reply:
@npretto I also added https://github.com/metabase/metabase/pull/49591/commits/ae414b3a9c4844676bb70026852bd407da92c6ac - so that it does not bail out if your tables do not have any multi-tenancy column, and just continues normally without multi-tenancy.